images/scripts/ostree.setup

```diff
@@ -37,6 +37,11 @@ if [ "$IMAGE" == "rhel4edge" ]; then
     # HACK: Fix .ssh ownership, see https://issues.redhat.com/browse/THEEDGE-3033
     chown admin:admin /var/home/admin/.ssh
 
+    # change how sudo is configured inside edge images via osbuild.ks, do not use passwordless wheel group
+    # but use passwordless admin user directly.
+    rm -f /etc/sudoers.d/%wheel
+    echo -e "admin\tALL=(ALL)\tNOPASSWD: ALL" >> /etc/sudoers
```

Comment:
What's the motivation here? I don't like this -- we should stick to what the OS does, i.e. test images that are as close to reality as possible. If it's a workaround for a bug, then it needs to be reported, and this should get a "# HACK: url" comment.

Reply:
Hi @martinpitt, It is the issue as we've seen that some sudo tests failed. and we do not know why. 
The root cause were that in previous version by kickstart just admin user were set as passwordless sudo user. In current images whole wheel group is in sudoers and as I've seen inside cockpit tests, they expect to have set `admin` user inside sudoers.

So as you said, it is hack, but works, the most proper solution would be:
 - to avoid using `wheel` group anywhere inside tests, and create own group, because some tests uses adding users into wheel to check it is able to do admin task but with password. 
 - and also be more robust how sudo for `admin` user is configured. do not rely that just `admin` is passwordless sudoer.
 
So it seems like bigger refactoring. My gut feeling here is  that it will need more work and changes inside lots of (>10 tests, maybe much more) to make it more robust. So I prefer to do some Hackish solution and maybe create issue to refactor and robustify using users and groups inside cockpit tests.

Reply:
> do not rely that just admin is passwordless sudoer.

We don't actually *mean* to assume that -- all our images are supposed to have "admin:foobar" and the same password for sudo. But indeed I just checked that the current rhel4edge image also has passwordless sudo, and an explicit entry in `/etc/sudoers`.

So let's keep that for now, and let's clean this up in a follow-up. This is a needless complication, but indeed not the fault of this PR. Sorry, didn't check before!